pyproject.toml

```diff
@@ -20,10 +20,10 @@ license = {file = "LICENSE.txt"}
 dependencies = [
     "pydantic>=1.9,<3",
     "coloredlogs>=14.0,<15.1",
-    "arelle-release>=2.3,<2.13",
+    "arelle-release>=2.3,<3",
     "frictionless>=4.4,<5",
     "sqlalchemy>=1.4,<3",
-    "pandas>=1.5,<2",
+    "pandas>=1.5,<2.1",
```

Comment:
Pandas 2.1 has already been released and I'd like to work on getting it working in the main PUDL repo, so could we set this to `pandas>=1.5,<2.2" instead?  Until we get it all working in PUDL the more restrictive version over there will be controlling.

Reply:
Sure, that's easy enough!